manifests/dev/vsphere-7.0/vanilla/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
Why not move this to deployment.yaml itself?

Reply:
The FVT team needs to use this first, create a secret in `vmware-system-csi` namespace and then finally use the `vsphere-csi-driver.yaml` to create the CSI entities. Therefore, kept it separate.